### PR TITLE
create login page MVP 

### PR DESCRIPTION
**Pull request by helen-chu**
Correctly returns cookie with email address and TA status, fails to redirect to home page after submission.

### CODE REVIEW

File: src/main/webapp/login.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <script src="login_script.js"></script>
+  </head>
+  <body>
+    <div id="login_form">
+        <h3>Log in below:</h3>
+        <form id="signup">
+            <label for="email">Email:</label><br>
+            <input type="text" id="email" name="email"><br>
+            <label for="pwd">Password:</label><br>
+            <input type="text" id="pwd" name="pwd"><br><br>
```
Comment (rlara-sps) [inline src/main/webapp/login.html]: This type should be `password`, so it displays: `*****`.

File: src/main/webapp/login.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            <input type="submit" value="Login" onclick="putCookie(document.getElementsByTagName('form'));">
```
Comment (rlara-sps) [inline src/main/webapp/login.html]: This action should be done in the Server. Refer to Week 2 walkthrough.

1. Remove this onclick action.
2. Create a Servlet for login, e.g. `LoginServlet.java`. The Servlet should implement `doPost()`. Here, you should extract and validate the email, i.e., decide if it corresponds to a TA. See Step 4 of walkthrough.
3. The Server sends cookies using `response.addCookie(...)`. See https://beginnersbook.com/2013/05/servlet-cookies/
4. Redirect is done with `response.sendRedirect("/index.html");`. See Step 8 of walkthrough.
5. Connect this form with the Servlet using `method` and `action`. See Step 5 of walkthrough.

**Review by rlara-sps — CHANGES_REQUESTED** [review]

File: src/main/webapp/login.html (hunk 1 above)
Comment (helen-chu) [reply src/main/webapp/login.html]: updated in newest commit

File: src/main/webapp/login.html (hunk 2 above)
Comment (helen-chu) [reply src/main/webapp/login.html]: updated in newest commit

**Review by helen-chu — COMMENTED** [review]
Updated in newest commit

File: src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+@WebServlet("/login-handler")
+public class LoginServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    Set<String> TAs = new HashSet<String>();
+    TAs.addAll(Arrays.asList(
+        new String[] {"TA1", "TA2"}));
```
Comment (rlara-sps) [inline src/main/java/com/google/sps/servlets/LoginServlet.java]: Nit: These are supposed to be emails, so I recommend more realistic examples, e.g. jbiden@email.com or hchu@sps.com

**Review by rlara-sps — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 3 above)
Comment (helen-chu) [reply src/main/java/com/google/sps/servlets/LoginServlet.java]: Updated in newest commit

**Pull request merged**